corehq/apps/userreports/tests/test_choice_provider.py

```diff
@@ -133,49 +137,80 @@ def test_get_choices_for_values(self):
         pass
 
 
-class LocationChoiceProviderTest(TestCase, ChoiceProviderTestMixin):
+class LocationChoiceProviderTest(ChoiceProviderTestMixin, LocationHierarchyTestCase):
     domain = 'location-choice-provider'
-
-    @classmethod
-    def make_location(cls, location_code, location_name, domain=None):
-        domain = domain or cls.domain
-        return make_loc(location_code, location_name, type='outlet', domain=domain)
+    location_type_names = ['state', 'county', 'city']
+    location_structure = [
+        ('Massachusetts', [
+            ('Middlesex', [
+                ('Cambridge', []),
+                ('Somerville', []),
+            ]),
+            ('Suffolke', [
+                ('Bostone', []),
```

Comment:
Yeah... this was a function of how these tests were set up. All the other tests that are "return all results" just search for the letter 'e' since that was assumed to be in every item in the search corpus...